README.md

```diff
@@ -1,215 +1,55 @@
-# Introduction to Apache Doris (incubating)
+# Apache Doris (incubating) Project
 
-Apache Doris is an MPP-based interactive SQL data warehousing for reporting and analysis. Doris mainly integrates the technology of Google Mesa and Apache Impala. Unlike other popular SQL-on-Hadoop systems, Doris is designed to be a simple and single tightly coupled system, not depending on other systems. Doris not only provides high concurrent low latency point query performance, but also provides high throughput queries of ad-hoc analysis. Doris not only provides batch data loading, but also provides near real-time mini-batch data loading. Doris also provides high availability, reliability, fault tolerance, and scalability. The simplicity (of developing, deploying and using) and meeting many data serving requirements in single system are the main features of Doris.
+Apache Doris is an MPP-based interactive SQL data warehousing for reporting and analysis. It open-sourced by Baidu. Please visit [Doris official site ](http://doris.incubator.apache.org) for more detail information, as well as [Wiki](https://github.com/apache/incubator-doris/wiki) for documents of install, deploy, best pracitices and FAQs.
```

Comment:
What about adding some ‘agitative’ description, or a brief of best Doris use case? like [kafka](https://kafka.apache.org/) or [druid](http://druid.io/)

Reply:
Our official website is not ready for users, there is nothing on it now. So here is our ground now.